# **AVOCADO DATA PROJECT**

## **_Overall analysis of avocado market in the US from 2015 to 2022_**

### <span style="font-size: 14px;">Analysis based on</span> **#avocado\_isUS\_2022**

1. **REVENUE**

In [1]:
--Conventional
SELECT year,AVG(average_price)*SUM(total_volume) as Revenue_Con FROM avocado_isUS_2022
WHERE type ='Conventional'
GROUP BY geography,type,year
ORDER BY year
--Organic
SELECT year,AVG(average_price)*SUM(total_volume) as Revenue_Org FROM avocado_isUS_2022
WHERE type ='Organic'
GROUP BY geography,type,year
ORDER BY year

(8 rows affected)

(8 rows affected)

Total execution time: 00:00:00.025

year Revenue_Con 2015 1359187051.8134363 2016 1308968540.5572515 2017 1494987598.2972066 2018 1380332178.0031629 2019 1362530490.0921106 2020 1174958156.7824662 2021 1025208166.8200291 2022 1058908312.2300941

year Revenue_Org 2015 36504230.53344666 2016 36241321.90639084 2017 38534030.991121165 2018 37019208.253344364 2019 29535711.58030229 2020 21413517.61187839 2021 17029318.6843664 2022 25004157.937022857

The data of Conventional & Organic avocado shows that avocado prices are fall sharply starting in 2020, which is most likely due to the impact of the COVID-19 epidemic. However, until 2022, sales of both avocados are showing a slight increase, which means that the avocado market in the US is showing signs of recovery.

The **Revenues** of Organic shows signs of stronger growth than the Conventional in 2022

**Conventional** in **2021** compared to **2022 :** an increase of about **3.29%**

**Organic** in **2021** compared to **2022 :** an increase of about **46.83%**

Need to analyze **PRICE** and **VOLUME** to understand why is there this different.

2. **PRICE**

In [2]:
--Conventional
SELECT  year, AVG(average_price) as Price_Con FROM avocado_isUS_2022
WHERE type ='Conventional'
GROUP BY geography,type,year
ORDER BY year
--Organic
SELECT  year, AVG(average_price) as Price_Org FROM avocado_isUS_2022
WHERE type ='Organic'
GROUP BY geography,type,year
ORDER BY year

(8 rows affected)

(8 rows affected)

Total execution time: 00:00:00.025

year Price_Con 2015 1.0124999915178006 2016 1.0467307682220752 2017 1.2216981177060109 2018 1.083673474740009 2019 1.143653848996529 2020 1.0187499982615311 2021 1.0942307733572447 2022 1.282307689006512

year Price_Org 2015 1.5023076901069055 2016 1.480384613458927 2017 1.6515094311732166 2018 1.5616326453734417 2019 1.6201923145697668 2020 1.5304166749119759 2021 1.6039583384990692 2022 1.7082692315945258

In the US, the average price of **Organic** avocados is always **higher** than **Conventional 33% - 50%** in during 7 years from **2015** to 2022.

The year **2022** is on record as the year with the **highest** avocado **prices** in both **Conventional** and **Organic.**

Rising prices mean that the US avocado market is facing the biggest **shortage of avocados** since 2015 so far. [(LINK)](https:\moit.gov.vn\tin-tuc\quoc-te\gioi-thieu-thi-truong-qua-bo-hoa-ky.html)

3. **VOLUME**

In [3]:
--Conventional
SELECT year, SUM(total_volume) as Volume_Con_lbs FROM avocado_isUS_2022
WHERE type ='Conventional'
GROUP BY geography,type,year
ORDER BY year
--Organic
SELECT year, SUM(total_volume) as Volume_Org_lbs FROM avocado_isUS_2022
WHERE type ='Organic'
GROUP BY geography,type,year
ORDER BY year

(8 rows affected)

(8 rows affected)

Total execution time: 00:00:00.026

year Volume_Con_lbs 2015 1342406976 2016 1250530299 2017 1223696408 2018 1273752851 2019 1191383644 2020 1153333162 2021 936921344 2022 825783329

year Volume_Org_lbs 2015 24298771 2016 24481017.6875 2017 23332613.34375 2018 23705452.34375 2019 18229756.625 2020 13991952.625 2021 10617057.984375 2022 14637129.484375

The sales volume of **Conventional** avocados **decreased** continuously for many years from **2019** (1,191,383,644/lbs) - **2022** (825,783,329/lbs).

**Organic** has a strong **increase** in sales volume in **2022** (14,637,129,484375/lbs) compared to the previous year's output in **2021** (10,617,057,984375/lbs).

It shows that customers in the US tend to prefer **Organic** avocado varieties over **Conventional** varieties.

  

**Specifically:**

In [4]:
SELECT year, 
SUM(_4046) as __4046Oz,
SUM(_4225) as __4225Oz,
SUM(_4770) as __4770Oz
FROM avocado_isUS_2022
WHERE type ='Organic'
GROUP BY geography,type,year
ORDER BY year

(8 rows affected)

Total execution time: 00:00:00.012

year __4046Oz __4225Oz __4770Oz 2015 9612079.96875 14376136.765625 310541.2287597656 2016 6934600.2578125 17116658.71875 429758.72705078125 2017 6882024.4296875 16282228.6875 157027.9600830078 2018 7104916.78125 15590364.515625 71548.4700012207 2019 6759083.984375 11218817.40625 247357.86840820312 2020 5799565.1484375 7950621.671875 240688.36840820312 2021 4119032.28125 6287378.484375 210647.24612426758 2022 5359734.7109375 4506033.0953063965 4771361.300014496

The unexpected increase in sales volume of **Organic** avocado varieties in **2022** is specifically due to the unexpected increase in sales of **\_4770** of **Organic** avocado varieties in **2022** (4,771,361,300014496) compared to 2021 only yes (210,647,24612426758)

In [5]:
SELECT year, 
SUM(_4770) as __4770Oz
FROM avocado_isUS_2022
WHERE type = 'Organic'
GROUP BY geography,type,year
ORDER BY year

(8 rows affected)

Total execution time: 00:00:00.013

year __4770Oz 2015 310541.2287597656 2016 429758.72705078125 2017 157027.9600830078 2018 71548.4700012207 2019 247357.86840820312 2020 240688.36840820312 2021 210647.24612426758 2022 4771361.300014496

Compared to **2021**, sales volume of **4770** **Organic** avocado varieties have increased **22 times** in **2022.**

In [7]:
--PERCENT OF ORGANIC VOLUME IN 2021
SELECT year, 
SUM(_4046)/SUM(total_volume)*100 as Per_4046Oz,
SUM(_4225)/SUM(total_volume)*100 as Per_4225Oz,
SUM(_4770)/SUM(total_volume)*100 as Per_4770Oz
FROM avocado_isUS_2022
WHERE type ='Organic' and year ='2021'
GROUP BY geography,type,year
ORDER BY year
--PERCENT OF ORGANIC VOLUME IN 2022
SELECT year, 
SUM(_4046)/SUM(total_volume)*100 as Per_4046Oz,
SUM(_4225)/SUM(total_volume)*100 as Per_4225Oz,
SUM(_4770)/SUM(total_volume)*100 as Per_4770Oz
FROM avocado_isUS_2022
WHERE type ='Organic' and year ='2022'
GROUP BY geography,type,year
ORDER BY year


(1 row affected)

(1 row affected)

Total execution time: 00:00:00.076

year Per_4046Oz Per_4225Oz Per_4770Oz 2021 38.79636230029008 59.219592599268665 1.9840453582741533

year Per_4046Oz Per_4225Oz Per_4770Oz 2022 36.61738947284006 30.784950697584147 32.597657246305566

Data show that the percentage of **\_4770** **Organic** avocados sold in **2022** has **increased** to **32.6%**, surpassing even **\_4225** **Organic (30.8%)**. While the same period last year, the sales volume of these two types of avocados had a large difference.

**4225 Organic (2021)**: accounting for **59.2%** of total organic sales volume  

**4770 Organic (2021)**: accounting for **1.98%** of total organic sales volume  

This shows that the trend of favoring **\_4770 Organic** avocado is appearing in the US. Demand for this type of avocado will also increase sharply in **2022.**

In [8]:
SELECT year, 
SUM(_4046) as __4046Oz,
SUM(_4225) as __4225Oz,
SUM(_4770) as __4770Oz 
FROM avocado_isUS_2022
WHERE type ='Conventional'
GROUP BY geography,type,year
ORDER BY year

(8 rows affected)

Total execution time: 00:00:00.013

year __4046Oz __4225Oz __4770Oz 2015 642295763.5 646812350 53298869.9375 2016 582012080.5 608520955.5 59997261.78125 2017 629495142.5 560752231.5 33449030.34375 2018 656826179.5 580248320.5 36678348.125 2019 615756300 530238781.5 45386245.625 2020 681665747 435489098.5 36082427.6875 2021 557177327 353250315.25 26493700.46875 2022 538637767.5 265609524.75 21536033.6875

For the **Conventional** avocado variety, the **downward** trend in avocado sales volume in all 3 types **\_4046,\_4225,\_4770** has not shown any signs of stopping.

It shows that customers are increasingly choosing **Organic** avocados over **Conventional** , even though Organic avocados are often more expensive.

3. **SEASON**

In [9]:
--SEASON_Conventional
SELECT MONTH(date) as Month, SUM(total_volume) as vol_con FROM avocado_isUS_2022
WHERE type ='Conventional'
GROUP BY geography,type,MONTH(date)
ORDER BY 1 
--SEASON_Organic
SELECT MONTH(date) as Month, SUM(total_volume) as vol_org FROM avocado_isUS_2022
WHERE type ='Organic'
GROUP BY geography,type,MONTH(date)
ORDER BY 1 

(12 rows affected)

(12 rows affected)

Total execution time: 00:00:00.026

Month vol_con 1 895064453 2 850165250 3 816241608 4 804253789 5 952383489 6 794186327 7 818447907 8 757567216 9 703507824 10 677362246 11 618353296 12 510274608

Month vol_org 1 13402442.9375 2 12777332.59375 3 14569741.984375 4 15184915.359375 5 18418215.546875 6 15811830.546875 7 12904256.265625 8 11816838.84375 9 11056192.25 10 10206592.296875 11 9828576.96875 12 7316815.5